pkg/formatter/table.go

```diff
@@ -200,5 +233,15 @@ func NodesReportSummary(records []*reporting.NodeReportItem, appliedNodesFilter
 	if len(appliedNodesFilter) > 0 {
 		fmt.Fprintf(buffer, appliedNodesFilterFmt, appliedNodesFilter)
 	}
+
 	return FormattedResult{buffer.String(), errMsg.String()}
 }
+
+func ShortFormat(name string) string {
+
+	if len(name) > 10 {
+		return name[0:9] + "..."
```

Comment:
```suggestion
		return name[0:10] + "..."
```

Reply:
Added a comment instead as we discussed.